### ISSUE TITLE
Build `tabula-py` for `osx-arm64` architecture?

### ISSUE THREAD

**Issue by zoltanmaric** [conda-forge/tabula-py-feedstock#22]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

On a 2020 MacBook Air with the Apple M1 chip, I can't install `tabula-py` with
```bash
conda install -c conda-forge tabula-py
```

Error message:

<details><summary>Error message</summary>

```bash
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - tabula-py

Current channels:

  - https://conda.anaconda.org/conda-forge/osx-arm64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

</details>

I believe the issue is that `tabula-py` is not built for the `osx-arm64` architecture (the M1 chips), but only for `osx-64` ( as the `tabula-py` Conda Platforms badge currently indicates: [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/tabula-py.svg)](https://anaconda.org/conda-forge/tabula-py))

I'm happy to help if I can to introduce a build for the `osx-64` platform.

### Installed packages

```shell
Irrelevant
```

### Environment info

```shell
Irrelevant
```

**Comment by zoltanmaric** [comment]
This should be resolved by #25.

**Comment by zoltanmaric** [comment]
Resolved with #25.
